front/components/app/DatasetView.tsx

```diff
@@ -351,18 +400,30 @@ export default function DatasetView({
 
     if (onUpdate) {
       // TODO(spolu): Optimize, as it might not be great to send the entire data on each update.
-      onUpdate(datasetInitializing, valid, {
-        name: datasetName.slice(0, MODELS_STRING_MAX_LENGTH),
-        // keys: datasetKeys,
-        description: (datasetDescription || "").slice(
-          0,
-          MODELS_STRING_MAX_LENGTH
-        ),
-        data: exportDataset(),
-      });
+      onUpdate(
+        datasetInitializing,
+        valid,
+        {
+          name: datasetName.slice(0, MODELS_STRING_MAX_LENGTH),
+          // keys: datasetKeys,
```

Comment:
Not sure I understand why this is commented out ?

Reply:
We don't pass keys there, only the name description and data. This was previously commented. Will remove